### PR TITLE
Create journal-week-1.md

### PR DESCRIPTION
**Pull request by rosalsm**

### CODE REVIEW

**Comment by jeffworley** [comment]
I enjoyed your journal-week-1.md Rosa!  It was concise and had a real positive tone.  If you find yourself with additional time and the urge to add a fold to your brain, I would suggest going into a bit more depth about what we learned on the broad topics you listed.  I found that doing this helped me to reinforce the knowledge. All in all :+1:

**Comment by boshnivolo** [comment]
I like your opening statement. We are still here! And the rest of your entry was good too. I'm glad that you've become more comfortable with GitHub (: :+1:

File: journal-week-1.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Week 1.
+
+We have learn a lot, we have struggled a lot too but we are still in here and looking forward.
```
Comment (kellymurray) [inline journal-week-1.md:3]: :+1:

File: journal-week-1.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+and the opposite when I need to start working with the files from remote to local (pull). Still tricky for me however 
+I can see improvements.
+
+I have learnt about JavaScript and its 4 primitive types (null, boolean, numbers and strings) and 2 contents (list or arrays
+dictionary as an object) and lots of tools to play with them.
+
+I learnt how to start thinking in order to solve a problem with code. Think about what variables to used, how and how to 
+get them related to the function.
+
+Now I have to share this with 2 more people.
```
Comment (kellymurray) [inline journal-week-1.md:21]: Thanks for sharing. I'd love to see you tackle bigger concepts as well, such as pair programming.

**Pull request merged**